assignments/array-methods.js

```diff
@@ -56,23 +56,57 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach(function(runners){
+    fullName.push(`${runners.first_name} ${runners.last_name}`);
+})
 console.log(fullName);
 
+
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+runners.map(function(runners){
```

Comment:
You don't need to push when using .map. You could've done the following: 

allCaps = runners.map(function (runner) {
    return runner.first_name.toUpperCase();
});

.map returns an array for us.